alfalfa_worker/step_sim/step_fmu.py

```diff
@@ -151,6 +151,7 @@ def run(self):
             while True:
                 message = self.redis_pubsub.get_message()
                 if message:
+                    print("process_pubsub_message: %s" % message)
```

Comment:
Do we really want print instead of logging here?

Reply:
the `step_fmu` guy doesn't have any logging infrastructure was the only reason I used a print.

Reply:
removed